from_cpython/Objects/stringobject.c

```diff
@@ -23,6 +23,22 @@ PyObject * _do_string_format(PyObject *self, PyObject *args, PyObject *kwargs) {
     return do_string_format(self, args, kwargs);
 }
 
+void _string_init() {
```

Comment:
I'm suggesting to just remove the ```_string_init``` func. Because it's getting executed after ```_PyUnicode_Init``` which will already have register this types.